mycore-base/src/main/java/org/mycore/frontend/cli/MCRAccessCommands.java

```diff
@@ -226,14 +227,13 @@ public static void exportAllPermissionsToFile(String filename) throws Exception
         out.output(doc, fos);
     }
 
-    private static Element getRuleFromFile(String fileName) throws Exception {
-        if (!checkFilename(fileName)) {
-            LOGGER.warn("Wrong file format or file doesn't exist");
-            return null;
-        }
-        Document ruleDom = MCRXMLParserFactory.getParser().parseXML(new MCRFileContent(fileName));
-        Element rule = ruleDom.getRootElement();
-        if (!rule.getName().equals("condition")) {
+    private static String filenameToUri(String filename) {
+        return "file://" + new File(filename).getAbsolutePath();
```

Comment:
I think it would be better to use the method: https://docs.oracle.com/javase/8/docs/api/java/io/File.html#toURI--